lib/tasks/data_hygiene.rake

```diff
@@ -19,4 +19,17 @@ namespace :data_hygiene do
       puts "#{owning_app}: #{without_content_id} without, #{with_content_id} with"
     end
   end
+
+  desc "Produce a CSV of artefacts with tags of a given type"
+  task list_tagged_artefacts: [:environment] do
+    csv_out = CSV.new($stdout)
+    csv_out << %w(content_id base_path format tag_type count)
+    artefacts = Artefact.where(owning_app: "whitehall", state: "live", :tags.ne => [])
+    artefacts.each do |artefact|
+      tag_type_counts = artefact.tags.group_by(&:tag_type).map { |tag_type, tags| [tag_type, tags.size] }
+      tag_type_counts.each do |tag_type, count|
+        csv_out << [artefact.content_id, artefact.slug, artefact.kind, tag_type, count]
```

Comment:
Offline discussed: remove `artefact.slug` & `artefact.kind`, use the content-store name of the tags to make diffing easier.